.gitignore

```diff
@@ -10,4 +10,7 @@ htmlcov/
 .venv/
 dist/
 .docs
-.vscode
\ No newline at end of file
+.vscode
+
+# virtual environment
+venv
```

Comment:
I'd rather this just go into your local gitignore since it's not part of any wider team workflow 😸  https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#excluding-local-files-without-creating-a-gitignore-file

Reply:
@i80and awesome thank you for that. Going to make these changes.